tests/test_F0_output.py

```diff
@@ -49,6 +49,27 @@
 
 BUS = pd.DataFrame({"timeseries 1": pandas_Series, "timeseries 2": pandas_Series})
 
+from src.constants import (
+    PLOTS_BUSSES,
+    PATHS_TO_PLOTS,
+    PLOTS_DEMANDS,
+    PLOTS_RESOURCES,
+    PLOTS_NX,
+    PLOTS_PERFORMANCE,
+    PLOTS_COSTS,
+)
+
+dict_plots = {
```

Comment:
I would tend to create such a `dict_plot` in `constants.py`

Reply:
Ah... yeah, makes sense. But don´t I need to use `.deepcopy()` then to make sure that the changes in eg. `dict_values `are not applied to the `dict_plots ` in the tests as well?

Reply:
```
d1 = dict(a=1, b=2)
d2 = dict(c=3)
d2.update(d1)
print(d2)
d1['d']=4
print(d2)
```
d2 is not modified, so probably `updates take care of that for you :)`